sdk/src/main/scala/io/horizen/certificatesubmitter/strategies/WithoutKeyRotationCircuitStrategy.scala

```diff
@@ -22,9 +23,10 @@ class WithoutKeyRotationCircuitStrategy[
   PM <: SidechainBlockBase[TX, H],
   HIS <: AbstractHistory[TX, H, PM, _, _, _],
   MS <: AbstractState[TX, H, PM, MS]](settings: SidechainSettings,
+                                      sc2scConfig: Sc2ScConfigurator,
                                       params: NetworkParams,
                                       circuit: ThresholdSignatureCircuit)
-  extends CircuitStrategy[TX, H, PM, HIS, MS, CertificateDataWithoutKeyRotation](settings, params) {
+  extends CircuitStrategy[TX, H, PM, HIS, MS, CertificateDataWithoutKeyRotation](settings, sc2scConfig, params) {
```

Comment:
revert all the changes in this class because it doesn't support sc2sc.
Circuit WITHOUT key rotation has no "free" custom fields for sc2sc needs.